packages/closer/pages/sales-terminal.tsx

```
@@ -0,0 +1,171 @@
+];
+
+const SalesTerminal: React.FC = () => {
+    const router = useRouter();
+    const { user, isAuthenticated } = useAuth();
+
+    // Redirect or show not found if not authenticated
+    if (!isAuthenticated || !user) {
+        return <PageNotFound error="Please log in to access the sales terminal." />;
+    }
+
+    const [selected, setSelected] = useState<MenuItem | null>(null);
```

ESLint

[error] React Hook "useState" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)